README.md

```diff
@@ -1,6 +1,48 @@
 # Job Manager
+[Development Readme](#development)
 
-Job Manager API and UI for interacting with asynchronous batch jobs.
+## What is the Job Manager?
+
+The Job Manager is an API and UI for monitoring and managing jobs in a backend execution engine. 
+
+The Broad, Verily, and many other organizations in the life sciences execute enormous numbers of scientific workflows and need to manage those operations. Job Manager was born out of the experiences of producing data for some of the world’s largest sequencing projects such as The Cancer Genome Atlas, Baseline, and the Thousand Genomes Project. 
+
+The Job Manager aspires to bring ease and efficiency to developing and debugging workflows while seamlessly scaling to production operations management.
+
+## Key Features
+* Sits on top of either [Cromwell](https://github.com/broadinstitute/cromwell) or [dsub](https://github.com/googlegenomics/dsub)
```

Comment:
nit: maybe adjust the wording slightly - current "either" seems too exclusive of other possible backends, maybe "Supports job visualization over Cromwell or dsub backends"?